lib/main.dart

```diff
@@ -42,3 +51,9 @@ class GlobeApp extends StatelessWidget {
         );
       });
 }
+
+Future<void> _firebaseInitializer() async => await Firebase.initializeApp(
```

Comment:
why Future ?

Reply:
Because we need to await for the firebase Initialisation before the setup of our InjectionDependencies. 
Otherwise, the remote config manager will try to be initiliazed before the "Firebase.initializeApp" method ends and the app will crash.